tests/lib/string_conv/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+tests:
+  libraries.string_conv:
+    tags: string to numeric conversion
+    platform_allow: native_posix
```

Comment:
this can be removed. We expect that it works on all platforms.

Reply:
Well, I expected that aswell but there are approximatley 10 platforms that fails the unit test. Most of these are related to QEMU platform. My thought was that as long as it runs on the native linux platform, it should cover most other platforms.